deployment/postgres-init-db/sql/types/permissions/permissions.sql

```diff
@@ -48,6 +48,7 @@ create type permissions.function_permission_key
     'get_conflicting_activities',
     'get_non_conflicting_activities',
     'get_plan_history',
+    'migrate_plan_to_model',
```

Comment:
MAJOR, BLOCKING: This change was not included in the migration (no, I don't know why the migration is passing with the enum definitions differing)

Reply:
fixed, maybe we can talk offline about better strategies for `enum`s since migrating them is painful